src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -61,7 +61,7 @@ export interface EpidemicModelInputs extends ModelInputsUpdate {
   rateOfSpreadFactor: RateOfSpread;
   usePopulationSubsets: boolean;
   facilityDormitoryPct: number;
-  facilityCapacity: number | undefined;
+  facilityCapacity?: number;
```

Comment:
you should be able to just remove this entirely; it's just duplicating what's in `ModelInputsPerisistent` (which this extends, by way of `ModelInputsUpdate`

Reply:
@macfarlandian Thanks for another look at this.

I had thought about removing it but noticed that this interface is described as 'some fields are required for calculations, define them here'. All of the other fields here are also defined in other interfaces inherited through `ModelInputsUpdate` so I figured this interface was more for documentation than actually defining anything new. I'll go ahead and remove it here, just curious about your thought process is about how to chose which fields remain defined in this interface vs via inheritance?

Reply:
FWIW, knowing what I know now about TypeScript, I wish I had defined all the inputs directly in `EpidemicModelInputs` and derived `ModelInputsUpdate` and `ModelInputsPersistent` via `Partial<>` and `Pick<>`. Same end result but I think it would be more readable and less error-prone.